benchmarks/benchmarks/fixed-poolifier.js

```diff
@@ -0,0 +1,27 @@
+// IMPORT LIBRARIES
+const { FixedThreadPool, DynamicThreadPool } = require('../../lib/index')
```

Comment:
You should not rely a the local build here but use the poolifier package in node_mobules. Otherwise the results are not replicable because not based on the very same code/version.

Reply:
Yes I agree, as I said this is done to test the performance enhancement that is into this PR, shall be removed for sure and tested with another beta